compute/admin_guide/authentication/active_directory.adoc

```diff
@@ -140,5 +140,5 @@ image::logout.png[width=200]
 . At Console's login page, enter the UPN and password of an existing Active Directory user.
 +
 If the log in is successful, you are directed to the view appropriate for the user's role.
-If you have the Access User role, you are directed to a single page, where you can download certs for xref:../access_control/rbac.adoc[Docker client role-based access control].
+//If you have the Access User role, you are directed to a single page, where you can download certs for xref:../access_control/rbac.adoc[Docker client role-based access control].
```

Comment:
we don't need this, as we removing this role

Reply:
Yes, so I commented these lines.